strawberryfields/backends/fockbackend/backend.py

```diff
@@ -290,3 +290,37 @@ def measure_fock(self, modes, shots=1, select=None, **kwargs):
                 "fock backend currently does not support " "shots != 1 for Fock measurement"
             )
         return self.circuit.measure_fock(self._remap_modes(modes), select=select)
+
+    def prepare_gkp(
+        self, state, epsilon, amplcutoff, representation="real", shape="square", mode=None
+    ):
+        r"""Prepares the Fock representation of a finite energy GKP state.
+
+        GKP states are qubits, with the qubit state defined by:
+
+        :math:`\ket{\psi}_{gkp} = \cos\frac{\theta}{2}\ket{0}_{gkp} + e^{-i\phi}\sin\frac{\theta}{2}\ket{1}_{gkp}`
+
+        where the computational basis states are :math:`\ket{\mu}_{gkp} = \sum_{n} \ket{(2n+\mu)\sqrt{\pi\hbar}}_{q}`.
+
+        Args:
+            state (list): ``[theta,phi]`` for qubit definition above
+            epsilon (float): finite energy parameter of the state
+            cutoff (float): this determines how many terms to keep
```

Comment:
Also, worth maybe clarifying the connection (or lack of) between `cutoff` used by the Fock backend and `amplcutoff`?
```suggestion
            amplcutoff (float): this determines how many terms to keep
```